config/default/manager_auth_proxy_patch.yaml

```diff
@@ -9,16 +9,6 @@ spec:
   template:
     spec:
       containers:
-      - name: kube-rbac-proxy
```

Comment:
I think commenting out the entire patch is a better idea https://github.com/kubevirt/ssp-operator/blob/master/config/default/kustomization.yaml#L31

Reply:
But we still need the manager container here

Reply:
The manager container will remain as it is present in the deployment base resource https://github.com/kubevirt/ssp-operator/blob/master/config/manager/manager.yaml#L25